.github/workflows/windows.yml

```diff
@@ -131,7 +131,7 @@ jobs:
         shell: pwsh
         run: |
           ${{ env.OV_INSTALL_DIR }}/setupvars.ps1
-          cmake -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -S ${{ env.SRC_DIR }} -B ${{ env.BUILD_DIR }}
+          cmake -DOpenVINODeveloperPackage_DIR=${{ env.OV_INSTALL_DIR }}/developer_package/cmake -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -S ${{ env.SRC_DIR }} -B ${{ env.BUILD_DIR }}
```

Comment:
looks like gtest become a dynamic library 
![{9D5D8850-768B-4676-9F48-E4AC09CA46A3}](https://github.com/user-attachments/assets/1b3ff01a-31cc-4c05-8661-3d66ab2ea94d)

I think we need to set it explicitly as static here:
https://github.com/openvinotoolkit/openvino.genai/blob/55ce655af15195f58a02d495650ec780af0a32ca/tests/cpp/CMakeLists.txt#L6-L13